package.json

```diff
@@ -34,32 +34,33 @@
     "access": "public"
   },
   "peerDependencies": {
-    "@callstack/repack": "blitzstudios/repack.git#callstack-repack-v2.6.3-gitpkg",
+    "@callstack/repack": "2.6.3",
     "@react-native-community/netinfo": "9.3.7",
     "axios": "0.15.3",
     "lodash": "4.17.21",
     "react": "17.0.2",
     "react-native": "0.66.5",
-    "react-native-linear-gradient": "2.5.6",
     "react-native-rename": "3.2.12",
-    "react-native-svg": "13.7.0",
-    "react-native-tcp-socket": "6.0.6",
-    "react-native-webview": "11.23.1"
+    "react-native-fast-image": "8.5.11",
+    "react-native-interactable": "0.1.10",
+    "react-native-linear-gradient": "2.5.6",
+    "react-native-svg": "13.7.0"
   },
   "devDependencies": {
     "@babel/core": "^7.12.9",
     "@babel/preset-typescript": "^7.16.7",
     "@babel/runtime": "^7.12.5",
     "@react-native-community/eslint-config": "^2.0.0",
     "@types/lodash": "^4.14.182",
-    "@types/react-native": "0.66.5",
+    "@types/react-native": "0.67.7",
```

Comment:
I believe this is intended to correspond to the current version of react native that we have. `0.66.5`. Do we need to update this?

Reply:
Oooh, I see. It was updated recently in the main app. Interesting.